package.json

```diff
@@ -3,10 +3,9 @@
   "description": "Netlify Node.js API client",
   "version": "6.1.8",
   "files": [
-    "src",
-    "dist",
-    "!*.test.js*",
-    "!*~"
+    "src/**/*.js",
+    "!src/**/*.test.js",
+    "dist/main.js"
```

Comment:
Do we want to publish `dist/main.js.map`?

Reply:
Good point. Fixed in 27f4f9f824970470df41194d355cb6bcf6adf5d6